sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs

```diff
@@ -18,37 +18,16 @@ internal static class TextAnalyticsServiceSerializer
 
         #region Serialize Inputs
 
-        public static ReadOnlyMemory<byte> SerializeDetectLanguageInputs(IEnumerable<string> inputs, string countryHint)
+        public static ReadOnlyMemory<byte> SerializeDetectLanguageInputs(IEnumerable<DetectLanguageInput> inputs, string defaultCountryHint)
         {
             var writer = new ArrayBufferWriter<byte>();
             var json = new Utf8JsonWriter(writer);
             json.WriteStartObject();
             json.WriteStartArray("documents");
-            int id = 1;
             foreach (var input in inputs)
             {
                 json.WriteStartObject();
-                json.WriteString("countryHint", countryHint);
-                json.WriteNumber("id", id++);
-                json.WriteString("text", input);
-                json.WriteEndObject();
-            }
-            json.WriteEndArray();
-            json.WriteEndObject();
-            json.Flush();
-            return writer.WrittenMemory;
-        }
-
-        public static ReadOnlyMemory<byte> SerializeDetectLanguageInputs(IEnumerable<DetectLanguageInput> inputs)
-        {
-            var writer = new ArrayBufferWriter<byte>();
-            var json = new Utf8JsonWriter(writer);
-            json.WriteStartObject();
-            json.WriteStartArray("documents");
-            foreach (var input in inputs)
-            {
-                json.WriteStartObject();
-                json.WriteString("countryHint", input.CountryHint);
+                json.WriteString("countryHint", input.CountryHint ?? defaultCountryHint);
```

Comment:
Nit - any chance an empty string would appear here that we'd also want to default?

Reply:
An empty string has a meaning -- it's the way the service has callers specify "None" (i.e. don't use a model trained for a certain country or language).  In our next preview, we're planning to move from strings to extensible enums, and will provide None there so it's more clear in our SDK what's happening.